lessons/10_debugging/classwork.md

```diff
@@ -0,0 +1,149 @@
+# Lesson 10 Classwork
+
+Let's practice debugging!
+
+Find the bugs!
+
+## Spec
+
+Create a simple web page for every exercise and fix the code.
+
+### Print numbers 1-10
+
+Expected Output: `"1 2 3 4 5 6 7 8 9 10"`
+
+```html
+<button onclick="button1()">Print numbers 1-10</button>
+```
+
+```js
+function button1() {
+  var string = ''
```

Comment:
`let` because you are mutating the variable

Reply:
and please use semicolon and the end. We want them to stick to that style.